test/JDBC/input/storedProcedures/Test-sp_helpsrvrolemember-vu-prepare.sql

```diff
@@ -0,0 +1,14 @@
+CREATE PROC test_sp_helpsrvrolemember_proc @rolename AS sys.SYSNAME = NULL
```

Comment:
to execute it as part of upgrade, it must be part of some schedule file, otherwise it is executing on latest only